docs/pgp-workaround.md

```diff
@@ -0,0 +1,58 @@
+# PGP fallback URL in air-gapped environment workaround
+
+## Reminders
+
+When Elastic Agent tries to perform an upgrade, it first verifies the binary signature with the key bundled in the Agent.
+This process has a backup mechanism that will use the key coming from https://artifacts.elastic.co/GPG-KEY-elastic-agent instead of the one it already has.
+
+In an air-gapped environment, the Agent won't be able to download the remote key and therefore cannot be upgraded.
```

Comment:
You should mentions the versions this affects, which would be all versions from the one where we introduced this up until we fix it.